Source/Scene/Scene.js

```diff
@@ -1310,6 +1456,15 @@ define([
 
             scene._globeDepth.executeCopyDepth(context, passState);
 
+            if (scene.debugShowGlobeDepth) {
```

Comment:
Ok. You'll need to make sure that the uniform state is updated to point to the right texture. Right now, I think it always points to the same texture or undefined if it isn't supported.